app/assets/stylesheets/retro.scss

```diff
@@ -1,41 +1,34 @@
-#nav {
+.nav {
   background-color: #696969;
   height: 60px;
+}
 
-  .headerIcons {
-    text-align: right;
+.headerItems {
```

Comment:
I can see that the names of the other declarations follows hyphen-separated case but this one follows camelCase. There is not a specific naming convention for class/id names, but is recommended to follow the chosen convention consistently 😃